packages/template-retail-react-app/jest-setup.js

```diff
@@ -38,22 +38,13 @@ export const setupMockServer = () => {
         rest.get('*/customers/:customerId/baskets', (req, res, ctx) =>
             res(ctx.delay(0), ctx.status(200), ctx.json(mockCustomerBaskets))
         ),
-        rest.post('*/customers/action/login', (req, res, ctx) => {
-            return res(
-                ctx.delay(0),
-                ctx.status(200),
-                ctx.json({
-                    authType: 'guest',
-                    customerId: 'customerid'
-                })
-            )
-        }),
         rest.post('*/sessions', (req, res, ctx) => res(ctx.delay(0), ctx.status(200))),
         rest.post('*/oauth2/token', (req, res, ctx) =>
             res(
                 ctx.delay(0),
                 ctx.json({
                     customer_id: 'customerid',
+                    // Is this token for guest or registered user?
```

Comment:
Does it matter? We [mock isTokenValid](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/6b9d4369b1460023de233e8b60b31feda38ebf55/packages/template-retail-react-app/jest-setup.js#L88-L94) to always be true, regardless of what the token is.

Reply:
Yes, so it's always valid during testing. But if we disable `bypassAuth`, then this mock would determine whether you're a guest or registered shopper initially, right?

Reply:
@wjhsf .The isTokenValid is for the old commerce-api, it does not make sense with the new integrations 
@vmarta If we disable `byPassAuth`, yes, it will go through the flow of guest unless we initialize a registered login in the test